meson.build

```diff
@@ -1398,7 +1393,7 @@ if features['dmabuf-interop-pl']
     sources += files('video/out/hwdec/dmabuf_interop_pl.c')
 endif
 
-features += {'dmabuf-wayland' : features['wayland'] and features['memfd_create'] and (features['vaapi'] or features['drm'])}
+features += {'dmabuf-wayland' : features['wayland'] and features['memfd_create'] and (features['vaapi-wayland'] or features['drm'])}
```

Comment:
Could you make this change a part of the commit before this one (the one that adds the new VO)?